doc/user_guide/performance_curves.rst

```diff
@@ -1,7 +1,7 @@
 .. _performance_curves:
 
-How are the performance curves sampled?
```

Comment:
As much as possible I'd avoid mixing refactorization and content changes, it makes the PR harder to review